include/openssl/service_indicator.h

```diff
@@ -28,56 +28,71 @@ OPENSSL_EXPORT uint64_t FIPS_service_indicator_after_call(void);
 
 OPENSSL_EXPORT const char* awslc_version_string(void);
 
-// |FIPS_service_indicator_check_approved| is intended to take in the before and
-// after counter values. It will return |AWSLC_APPROVED| if the approval check
-// was successful, and |AWSLC_NOT_APPROVED| if otherwise.
-//
-// Service indicator calls should not be used in non-FIPS builds. However, if
-// used, the direct check to |FIPS_service_indicator_check_approved|
-// will always indicate |AWSLC_APPROVED| in non-FIPS builds.
-// It is recommended to use the macro |CALL_SERVICE_AND_CHECK_APPROVED| though,
-// which will also always return |AWSLC_APPROVED| in non-FIPS builds.
-OPENSSL_EXPORT int FIPS_service_indicator_check_approved(uint64_t before, uint64_t after);
-
 #if defined(AWSLC_FIPS)
-#define AWSLC_MODE_STRING "AWS-LC FIPS "
 
-// This macro provides a bundled way to do an approval check and run the service.
-// The |approved| value passed in will change to |AWSLC_APPROVED| and
-// |AWSLC_NOT_APPROVED| accordingly to the approved state of the service ran.
-// It is highly recommended that users of the service indicator use this macro
-// when interacting with the service indicator.
-#define CALL_SERVICE_AND_CHECK_APPROVED(approved, func)             \
-  do {                                                              \
-    (approved) = AWSLC_NOT_APPROVED;                                \
-    int before = FIPS_service_indicator_before_call();              \
-    func;                                                           \
-    int after = FIPS_service_indicator_after_call();                \
-    if (FIPS_service_indicator_check_approved(before, after)) {     \
-        (approved) = AWSLC_APPROVED;                                \
-    }                                                               \
- }                                                                  \
- while(0)                                                           \
+#define AWSLC_MODE_STRING "AWS-LC FIPS "
```

Comment:
is the space at the end of the string a typo? the same for the non-FIPS string?

Reply:
No I believe it's actually intentional. It's used in `AWSLC_VERSION_STRING` and printing it along with the `AWSLC_VERSION_NUMBER_STRING` appended behind it will give "AWS-LC FIPS 1.1.0" and "AWS-LC 1.1.0".